multivac/gather_job_data.py

```diff
@@ -175,15 +185,24 @@ def put_to_db(self):
         for job in list(self.gathered_data.keys()):
             curr_job = self.gathered_data[job]
 
-            # Convert string to a datetime object, then convert it to unix
-            # time (seconds), then to timestamp in nanoseconds.
-            time_to_datetime = datetime.fromisoformat(
-                f"{curr_job['queued_at'].rstrip('Z')}+00:00")
-            time_to_unix = datetime.timestamp(time_to_datetime)
-            unix_ns_time = int(time_to_unix * 1000000000)
+            if 'failure_type' in curr_job.keys():
+                measurement = curr_job['failure_type']
+            else:
+                measurement = 'successful_jobs' \
+                    if curr_job['status'] == 'success' \
+                    else 'unknown_failure'
+
+            time_job_queued = github_time_to_unix(curr_job['queued_at'])
+            time_job_started = github_time_to_unix(curr_job['started_at'])
+            time_job_completed = github_time_to_unix(curr_job['completed_at'])
+
+            time_job_in_queue = time_job_started - time_job_queued
+            time_job_in_progress = time_job_completed - time_job_started
 
             tags = {
+                'job_id': curr_job['job_id'],
                 'job_name': curr_job['job_name'],
+                'run_id': curr_job['run_id'],
```

Comment:
to be exact, both are 'runs': job run and workflow run.

I propose naming it `workflow_id` or `workflow_run_id`

Reply:
done